Automattic/newspack-electionkit
# fix: correct escaping for twitter handle

### All Submissions:






### Changes proposed in this Pull Request:



Fixes a bug in the rendering of candidate Twitter URLs. The Twitter handle was being escaped as a URL which prepended a scheme, break the URL. 

### How to test the changes in this Pull Request:

1. On `master` branch, view a post with the plugin shortcode (`[sample_ballot]`). Input a zip code.
2. Observe the Twitter URLs are incorrect, e.g. `https://twitter.com/http://JoeBiden`.
3. Switch to this branch and repeat. Observe the Twitter URLs are correct, e.g. `https://twitter.com/JoeBiden`

### Other information:
